src/dymaptic.GeoBlazor.Core/ReadMe.md

```diff
@@ -82,11 +84,26 @@ ArcGIS JavaScript API, but without having to write a single line of JavaScript.
    @using dymaptic.GeoBlazor.Core.Components.Widgets
    @using dymaptic.GeoBlazor.Core.Objects
    ```
-6. In `Program.cs`, add the following line to your `builder.Services` to inject logic components like `GeometryEngine`.
+6. In `Program.cs` (for each executable project), add the following line to your `builder.Services` to inject logic components like `GeometryEngine`.
 
-```csharp
-   builder.Services.AddGeoBlazor();
-```
+   ```csharp
+      builder.Services.AddGeoBlazor();
+   ```
+
+   If you are using Blazor Server or InteractiveServer mode in Blazor Web Apps, you should also add the following lines
+   to `Program.cs` to support the `.wsv` file type.
+
+   ```csharp
+   var provider = new FileExtensionContentTypeProvider();
+   provider.Mappings[".wsv"] = "application/octet-stream";
+
+   app.UseStaticFiles();
+   // NOTE: for some reason, you still need the plain "UseStaticFiles" call above
+   app.UseStaticFiles(new StaticFileOptions
+   {
+       ContentTypeProvider = provider
+   });
+   ```
 
```

Comment:
Do we want to add a blurb about the licensing in this readme?

Reply:
The Registration logic is added on the Registration PR